evennia/objects/objects.py

```diff
@@ -1011,7 +1019,8 @@ def clear_contents(self):
             obj.move_to(home, move_type="teleport")
 
     @classmethod
-    def create(cls, key, account=None, **kwargs):
+    def create(cls, key: str, account: "DefaultAccount" = None, creator: "DefaultObject" = None, method: str = "create",
+               **kwargs):
```

Comment:
Naming is hard. 
- To be in line with other uses, I suggest using `caller` instead of `creator` here; we don't use something like a `creater ` anywhere else in Evennia. 
- I'm ambivalent on `method`. I suppose you may be inspired by `GET/POST/etc` methods here? I wonder if `source` may be a more general name, since this can also be called by the web component's views, but not sure :thinking:

Reply:
I can think of a few cases where the OWNER of an object might not be its creator, but... okay, will use caller, for now.

You're right on the odd naming choice for method, I'm not sure what else to call the darned thing. The hope is to make information about what sort of building is going on available to mechanisms that can make further decisions, and do more checks relevant to a game... I'm not sure how useful it is right this very moment, but I figured it may be useful to SOMEONE so it's better to have it than not...

Reply:
I think `source` probably works well, since the idea of it seems to be "where is this create call coming from", but if it doesn't actually have a specific purpose atm and is more of a "someone might want it someday" *and* you aren't even sure what to call it because the idea is too nebulous for a specific term, then maybe it's best kept in `kwargs`?

Reply:
I don't really mind having the ability to customize this, but yeah, maybe `source` could be a better name.